metapool/scripts/seqpro.py

```diff
@@ -73,10 +73,9 @@ def format_preparation_files(run_dir, sample_sheet, output_dir, verbose):
         df.to_csv(fp, sep='\t', index=False)
 
         if verbose:
-            project_name = remove_qiita_id(project)
             # assume qiita_id is extractable and is an integer, given that
             # we have already passed error-checking.
-            qiita_id = project.replace(project_name + '_', '')
+            qiita_id = get_qiita_id_from_project_name(project)
```

Comment:
This line will only be ran if we are in verbose mode, right? Should it be out of the if?

Reply:
@antgonza again, I cannot speak to that.  The `get_qiita_id_from_project` call is just a drop-in replacement for the original string `replace` call; I explicitly did NOT change any logic.  I'm not sure who wrote this module, but I suspect @charles-cowart  is the expert here.

Reply:
no for this particular case verbose is intended. I do have a PR waiting for review that addresses seqpro but it's awaiting review. ;)